src/options.ts

```diff
@@ -109,6 +109,36 @@ const parseOrUndefinedList = <T>(key: string, schema: z.ZodSchema<T>): T[] | und
     .filter((s) => s !== '')
 }
 
+const determineMicromambaInstallation = (micromambaBinPath?: string, downloadMicromamba?: boolean) => {
+  const preinstalledMicromamba = which.sync('micromamba', { nothrow: true })
+  if (preinstalledMicromamba) {
+    core.info(`Found preinstalled micromamba at ${preinstalledMicromamba}`)
+  }
+
+  if (downloadMicromamba === false && !preinstalledMicromamba) {
+    throw new Error('Could not find a pre-installed micromamba installation and `download-micromamba` is false.')
+  }
+
+  if (micromambaBinPath) {
```

Comment:
This confuses me. What's the difference between `micromambaBinPath` and `PATHS.micromambaBin`? Are we not downloading micromamba to `micromambaBinPath`?

Also, if I'm seeing things correctly, things branch could be moved to the very top of the function to make things more obvious

Reply:
`PATHS.micromambaBin` is the default value if `micromambaBinPath` is not defined.

If `micromambaBinPath` is defined and passed by the user we assume micromamba already exists at this location and use it.